load_locales! causes `may not live long enough` error

I have a Tauri App project with Leptos as frontend. I follow the book and add the macro to the `main.rs` but get the 'may not live long enough' error.

```
mod app;

use leptos::prelude::mount_to_body;
use leptos::*;
use thaw::{ConfigProvider, ToasterProvider};

use app::*;

leptos_i18n::load_locales!();
use i18n::*;

fn main() {
    console_error_panic_hook::set_once();
    mount_to_body(|| {
        view! {
            <I18nContextProvider>
                <ConfigProvider>
                    <ToasterProvider>
                        <App/>
                    </ToasterProvider>
                </ConfigProvider>
            </I18nContextProvider>
        }
    })
}
```

The full error logs are:

```
error[E0310]: the parameter type `__into_view_comp_b__` may not live long enough
 --> src/main.rs:9:1
  |
9 | leptos_i18n::load_locales!();
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  | |
  | the parameter type `__into_view_comp_b__` must be valid for the static lifetime...
  | ...so that the type `__into_view_comp_b__` will meet its required lifetime bounds
  |
  = note: this error originates in the macro `$crate::__private::macros_reexport::load_locales` which comes from the expansion of the macro `leptos_i18n::load_locales` (in Nightly builds, run with -Z macro-backtrace for more info)
help: consider adding an explicit lifetime bound
 --> /home/...../.cargo/registry/src/index.crates.io-6f17d22bba15001f/leptos_i18n-0.5.2/src/macros.rs:10:67
  |
10|         $crate::__private::macros_reexport::load_locales!{$($tt)*}: 'static
```

Am I doing wrong?

--- a/leptos_i18n/src/macro_helpers/interpol_args.rs
+++ b/leptos_i18n/src/macro_helpers/interpol_args.rs
@@ -6,13 +6,13 @@ pub trait InterpolateVar: IntoView + Clone + 'static + Send + Sync {}
 impl<T: IntoView + Clone + 'static + Send + Sync> InterpolateVar for T {}
 
 /// Marker trait for a type that can be used as an interpolation component.
-pub trait InterpolateComp<O: IntoView + Clone>:
+pub trait InterpolateComp<O: IntoView + Clone + 'static>:
     Fn(leptos::children::ChildrenFn) -> O + Clone + 'static + Send + Sync
 {
 }
 
 impl<
-        O: IntoView + Clone,
+        O: IntoView + Clone + 'static,
         T: Fn(leptos::children::ChildrenFn) -> O + Clone + 'static + Send + Sync,
     > InterpolateComp<O> for T
 {

--- a/leptos_i18n_macro/src/load_locales/interpolate.rs
+++ b/leptos_i18n_macro/src/load_locales/interpolate.rs
@@ -135,7 +135,7 @@ impl Field {
             VarOrComp::Comp { into_view } => {
                 let ts = [
                     quote!(#generic: l_i18n_crate::__private::InterpolateComp<#into_view>),
-                    quote!(#into_view: l_i18n_crate::reexports::leptos::IntoView + Clone),
+                    quote!(#into_view: l_i18n_crate::reexports::leptos::IntoView + Clone + 'static),
                 ];
                 EitherIter::Iter2(ts.into_iter())
             }
@@ -682,7 +682,7 @@ impl Interpolation {
             return quote! {
                 #[allow(non_camel_case_types)]
                 impl<#(#left_generics,)*> #ident<#(#right_generics,)*> {
-                    pub fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone {
+                    pub fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone + 'static {
                         let _ = self;
                         #key
                     }
@@ -699,7 +699,7 @@ impl Interpolation {
             quote! {
                 #[allow(non_camel_case_types)]
                 impl<#(#left_generics,)*> #ident<#(#right_generics,)*> {
-                    pub async fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone {
+                    pub async fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone + 'static {
                         #destructure
                         match #locale_field {
                             #(
@@ -713,7 +713,7 @@ impl Interpolation {
             quote! {
                 #[allow(non_camel_case_types)]
                 impl<#(#left_generics,)*> #ident<#(#right_generics,)*> {
-                    pub fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone {
+                    pub fn into_view(self) -> impl l_i18n_crate::reexports::leptos::IntoView + Clone + 'static {
                         #destructure
                         match #locale_field {
                             #(

